client/src/components/Banner.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react'
+import { Box } from 'grommet'
+import { Icon } from 'components/Icon'
+import styled, { css } from 'styled-components'
+
+export const Banner = ({
+  bgColor = 'brand',
+  fontColor = 'white',
+  iconName,
+  iconSize,
+  children
+}) => {
+  const [showing, setShowing] = useState(true)
+  const BannerBox = styled(Box)`
```

Comment:
I think that a lot of this can be done with grommet using `Text`, `Paragraph`, and `Box` attributes. Please try and move what is possible outside of custom css. Sometimes it's easier to layout text by wrapping it in a `Box`

Reply:
Inline styles are now updated as Grommet props(note: CSS rule for `children` props will remain as inline). Thank you!